cmd/jujuc/main_test.go

```diff
@@ -1,207 +0,0 @@
-// Copyright 2012, 2013 Canonical Ltd.
-// Licensed under the AGPLv3, see LICENCE file for details.
-
-package main
-
-import (
-	"bytes"
-	"errors"
-	"flag"
-	"fmt"
-	"io"
-	"os"
-	"os/exec"
-	"path/filepath"
-	stdtesting "testing"
-
-	"github.com/juju/cmd/v3"
-	"github.com/juju/gnuflag"
-	"github.com/juju/loggo"
-	jc "github.com/juju/testing/checkers"
-	gc "gopkg.in/check.v1"
-
-	jujucmd "github.com/juju/juju/cmd"
-	"github.com/juju/juju/juju/sockets"
-	coretesting "github.com/juju/juju/testing"
-	"github.com/juju/juju/worker/uniter/runner/jujuc"
-)
-
-func TestPackage(t *stdtesting.T) {
-	// TODO(waigani) 2014-03-19 bug 1294458
-	// Refactor to use base suites
-
-	coretesting.MgoSSLTestPackage(t)
-}
-
-type MainSuite struct{}
-
-var _ = gc.Suite(&MainSuite{})
-
-var flagRunMain = flag.Bool("run-main", false, "Run the application's main function for recursive testing")
-
-// Reentrancy point for testing (something as close as possible to) the jujud
-// tool itself.
-func TestRunMain(t *stdtesting.T) {
```

Comment:
Why don't we need these tests anymore?

Reply:
You need dqlite and cgo in here to run these... We'll need to move these to integration tests. Maybe @jack-w-shaw can have a look at these?